dev/game.py

```diff
@@ -88,14 +93,14 @@ def move_all(self, direction, distance,roll_over=False, replace="empty"):
                 spr.x += distance
 
     #Move one sprite, given distance, currently goes 1 step at a time, may want to update this?
-    def move_sprite(self, sprite, dx, dy, stop=True, replace="empty", roll_over=False, pass_through=False):
+    def move_sprite(self, sprite, dx, dy, stop=True, replace="empty", roll_over=False, kill=False,border = False):
```

Comment:
Minor style issue with spacing at end of line.
Could it be changed to `kill=False, border=False`?